src/Nethermind/Nethermind.Consensus/Tracing/Tracer.cs

```diff
@@ -16,18 +16,20 @@ public class Tracer : ITracer
         private readonly IWorldState _stateProvider;
         private readonly IBlockchainProcessor _traceProcessor;
         private readonly IBlockchainProcessor _executeProcessor;
-        private readonly ProcessingOptions _processingOptions;
+        private readonly ProcessingOptions _executeOptions;
+        private readonly ProcessingOptions _traceOptions;
 
         public Tracer(IWorldState stateProvider, IBlockchainProcessor traceProcessor, IBlockchainProcessor executeProcessor,
-            ProcessingOptions processingOptions = ProcessingOptions.Trace)
+            ProcessingOptions executeOptions = ProcessingOptions.Trace, ProcessingOptions traceOptions = ProcessingOptions.Trace)
```

Comment:
Shouldn't `Execute` be always `Trace`
And `Trace` be always `TraceTransaction`
And we don't need the parameters?